posts/2021-03-19-microprofile40-open-liberty-21003.adoc

```diff
@@ -440,7 +442,10 @@ MicroProfile JWT 1.2 simplifies the configuration for managing the validation of
 
 MicroProfile JWT 1.2 adds a number of MicroProfile Config properties to control different aspects of the JWT validation. The specification also adds support for the ES256 signature algorithm, while this Open Liberty beta supports using the RS384, RS512, HS384, HS512, ES256, ES384, and the ES512 signature algorithms.
 
-MicroProfile JWT 1.2 also supports the JWT specified in a cookie.
+
+Check out the following blog on MicroProfile JWT 1.2 to learn all of the exciting features.
+
+* link:{url-prefix}/blog/2021/03/24/whats-new-in-MP-Rest-Client2.0.html[Blog: MicroProfile JSON Web Token 1.2]
```

Comment:
The JWT blog lives here https://openliberty.io/blog/2021/03/26/MP-JWT-1.2.html. The above url seems not quite in sync with the real url.

Reply:
Changed to correct link